File: amypo/java/6-Functions/workout/funw-1.java
import java.util.Scanner;
public class P
{
    public static void main(String[]args)
    {
        Scanner sc=new Scanner(System.in);
        String bin_str=sc.nextLine().trim();
        if(!bin_str.matches("[01]+"))
        {
            System.out.println("Invalid input");
        }
        else
        {
        int deci=Integer.parseInt(bin_str,2);
        System.out.println(deci);
        }
        
    }
}

File: amypo/python/5-Functions/workout/funw-1.py
def i_d(n):
    total=0
    while(n>0):
        digit=n%10
        total+=digit
        n//=10
    return total
try:
    n=int(input())
    if n<0:
        print("Invalid Input")
    else:
        print(i_d(n))
except:
    print("Invalid Input")